### ISSUE TITLE
Timeline: Accessibility issues

### ISSUE THREAD

**Issue by lv75** [SAP/ui5-webcomponents-react#1758]
**Describe the bug**
As shown in the screenshots, the React UI5 Timeline and TimelineItem components have missing or misplaced ARIA role attributes, which are applied to the shadow DOM elements and cannot be changed. But they do affect accessibility.

**Expected behavior**
Can you please check, and fix or provide the details how we can avoid these A11y issues.
Thanks.

**Screenshots**
![Screen Shot 2021-06-22 at 7 22 20 PM](https://user-images.githubusercontent.com/8533257/123027887-64d0f080-d393-11eb-81c6-5a8ab3b102a4.png)
![Screen Shot 2021-06-22 at 7 23 52 PM](https://user-images.githubusercontent.com/8533257/123027971-803bfb80-d393-11eb-9683-f11ce5b8b289.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 0.31.9
`@ui5/webcomponents-react` version: 0.131.9
Operating System: MacOS
Browser: FF, Chrome et all.

**Comment by MarcusNotheis** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
